rebar.config

```diff
@@ -7,7 +7,7 @@
 
 {deps, [
     {protobuffs, "0.8.*", {git, "git://github.com/basho/erlang_protobuffs.git", {tag, "0.8.2"}}},
-    {hamcrest, ".*", {git, "https://github.com/hyperthunk/hamcrest-erlang.git", {branch, "master"}}}
+    {hamcrest, ".*", {git, "https://github.com/basho/hamcrest-erlang.git", {tag, "0.2.0-basho"}}}
```

Comment:
Well actually:

```
==> riak_pb (dialyzer)
Unknown types:
  hamcrest:matchspec/0
make: *** [tools.mk:39: dialyzer-run] Error 1
```

Seems like a really lame reason to pull it in, but there it is.